### DISCUSSION TITLE
Is this compatible with semantic-react and SSR?

### DISCUSSION THREAD

**Discussion by fullofcaffeine** [fomantic/Fomantic-UI#983] [General] [answered]
Hi there!

Found this after seeing the semantic-ui repo has been stagnant for almost an year already. Looks promising!

I'm about to start a greenfield project that uses React and requires some SSR. I'm wondering if this can be a drop-in replacement for `semantic-ui` in `semantic-ui-react`, somehow? Or is there such a thing like `fomantic-ui-react`? Also, does it support SSR out-of-the-box?

Thanks in advance!

**Comment by exoego** [comment]
Duplicates of https://github.com/fomantic/Fomantic-UI/issues/596

> For now you could use Fomantic-UI together with Semantic UI react. Here is a guide
> https://medium.com/@fomantic/setting-up-fomantic-ui-with-react-f701b68f736c

**Comment by exoego** [comment, marked as answer]
I think SSR support  is semantic-ui-react's concern.
FUI do not consider SSR support right now.

**Comment by fullofcaffeine** [comment]
Thanks!

**Comment by naterexw** [comment]
Here is a Next.js + Fomantic-UI + React starter template
https://github.com/skydiver/nextjs-semantic
